pantheon.gitignore

```diff
@@ -0,0 +1,33 @@
+###
```

Comment:
It looks like you added this pantheon.gitignore file. Can you explain how/why this file is added here?

Reply:
Noticed the urban-main-site had a pantheon.gitignore file as part of its project structure and was under the impression that it may be applicable when specifying untracked files for pantheon to ignore.

Reply:
Did you add it from urban.org? Every project can be different, so I wouldn't add until it is required.